templates/users/show.html

```diff
@@ -0,0 +1,35 @@
+{% extends "includes/base.html" %}
+{% load follower_tags %}
+{% block main_content %}
+<a href="{% url 'home' %}" style="float: right">Home</a>
+<h4>User Details</h4>
+<div>
+  <h6>First Name - {{ user.first_name }}</h6>
+  <h6>Last Name - {{ user.last_name }}</h6>
+  <h6>Followers - {{ user.followers.count }}</h6>
+  {% if request.user|can_follow:user %}
+    <a href="javascript:void(0)" onclick="follow('POST')">Follow</a>
+  {% endif %}
+  {% if request.user|can_unfollow:user %}
+  <a href="javascript:void(0)" onclick="follow('DELETE')">Unfollow</a>
+  {% endif %}
```

Comment:
these should be just `is_following` and then you can do 
`request.user != user and request.user|is_following:user`
`request.user != user and !request.user|is_following:user`